docs/introduction/developer-quickstart.md

```diff
@@ -2,7 +2,114 @@
 sidebar_position: 4
 ---
 
-# Developer Quickstart
+# Migrate your dApp to Obscuro
+
+## 1. Setting Up the Project
+
+This section will walk you through setting up a Hardhat project for Obscuro, installing plugin, and configuring network.
+
+### Prerequisites
```

Comment:
I don't think we need the detailed instructions on how  in this doc. 
 Let's move them to a different doc, and link to them.